docker-compose/docker-compose.yaml

```diff
@@ -28,6 +29,27 @@ services:
       PROMSCALE_DB_CONNECT_RETRIES: 10
       PROMSCALE_WEB_TELEMETRY_PATH: /metrics-text
       PROMSCALE_DB_URI: postgres://postgres:password@db:5432/postgres?sslmode=allow
+      PROMSCALE_ENABLE_FEATURE: tracing
+      PROMSCALE_OTLP_GRPC_SERVER_LISTEN_ADDRESS: ":9202"
+
+  otel-collector:
+    platform: linux/amd64
+    image: "otel/opentelemetry-collector:0.41.0"
+    command: [ "--config=/etc/otel-collector-config.yml" ]
+    volumes:
+      - ./otel-collector-config.yml:/etc/otel-collector-config.yml
+    ports:
+      - "4317:4317" # OTEL GRPC
+
+  tracegen:
```

Comment:
Sharing the same from the slack thread:

I’m personally against tracegen, As we already used it in past for testing trace ingestion, which didn’t actually help with real-world data or at least something similar. Tracegen was all about random string in the trace data model. This will not be useful for users anyhow to at least get the feel and visualize the traces.

I think the easiest way to generate some traces is to ingest traces from Promscale. Currently, we are already in the implementation phase of exposing traces from Promscale that should be the source of traces rather than going with some random trace generator into docker-compose. On the other side, node-exporter is a valid metrics generator from node so it adds value into docker-compose.